model/task_lifecycle.go

```diff
@@ -1421,19 +1404,35 @@ func evalLinearStepback(ctx context.Context, t *task.Task, caller string, newSte
 }
 
 // evalBisectStepback performs bisect stepback on the task.
-func evalBisectStepback(ctx context.Context, t *task.Task, caller string, newStepback, shouldStepback bool) error {
+func evalBisectStepback(ctx context.Context, t *task.Task, newStepback, shouldStepback bool) error {
 	// If the task is aborted or stepback is disabled then no-op.
 	if t.Aborted || !shouldStepback {
 		return nil
 	}
 
 	existingStepback := !t.StepbackInfo.IsZero() && t.ActivatedBy == evergreen.StepbackTaskActivator
-	generatedTask := t.GeneratedBy != ""
-	if newStepback || existingStepback || generatedTask {
+	generatedTaskStepback := t.GeneratedBy != ""
+	if newStepback || existingStepback || generatedTaskStepback {
```

Comment:
We don't want to go into stepback if `generatedTaskStepback` is the only thing that's true. Given the changes below, I think we should revert these two lines to what they were before.

Reply:
Yes! This was my bad